config/routes.rb

```diff
@@ -211,6 +211,7 @@
     get '/users/two_factor_authentication' => 'users/two_factor_authentication#show',
         as: :user_two_factor_authentication # route name is used by two_factor_authentication gem
     get '/backup_code_depleted' => 'users/backup_code_setup#depleted'
```

Comment:
Should we remove this route and the associated view?

Reply:
The view as well?

https://github.com/18F/identity-idp/blob/304b102ed74f6666042a3440fdd5ef8286f93ca0/app/views/users/backup_code_setup/depleted.html.erb